__tests__/contracts.js

```diff
@@ -999,3 +1000,48 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
     expect(checked.alreadyDeployed)
         .toBeTruthy();
 });
+test('do_tvm_transfer() should carry all the remaining balance of the current smart contract to another address', async () => {
+    const { contracts, crypto, queries } = tests.client;
+    const giverKeys = await crypto.ed25519Keypair();
+
+    const transferPackage = TransferContractPackage[2];
+    const contractData = await tests.deploy_with_giver({
+        package: transferPackage,
+        constructorParams: {},
+        keyPair: giverKeys,
+    });
+
+    const originalBalance = (await queries.accounts.waitFor(
+        {
+            id: { eq: contractData.address },
+        },
+        'balance',
+    )).balance;
+
+    // eslint-disable-next-line no-bitwise
+    const flags = 0 | 128;
```

Comment:
Why not `const flags = 128;`?

Reply:
Revert to 128 till fix for 0 |  32 | 128 test